README.md

```diff
@@ -28,6 +28,30 @@ As a pet sitter, you can provide the service of walks using your own schedule, p
 
 ## Installation
 
+### For version 0.4.0 and above
```

Comment:
Let's add this in the future if it'll be actual.
```suggestion
### For version 0.4.0
```

Reply:
Fixed with 4f56140